modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeJavaSetResultsSetTotalMethodCheck.testjava

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.source.formatter.dependencies.upgrade;
+
+/**
+ * @author Tamyris Bernardo
+ */
+public class UpgradeJavaSetResultsSetTotalMethodCheck {
+
+	public void method() {
+
+		SearchContainer searchContainer = new SearchContainer();
+
+
```

Comment:
We should remove the newline here.

You can do this by obtaining the method call's indentation using `JavaSourceUtil.getIndent()` and do something like this:

```java
String indent = JavaSourceUtil.getIndent(...);

String fullMethodCall = StringBundler.concat(
    StringPool.NEW_LINE, indent, methodCall, StringPool.SEMICOLON, StringPool.NEW_LINE);

newContent = StringUtil.replace(newContent, fullMethodCall, "");
```